Examples/OneSignalDemo/app/src/main/AndroidManifest.xml

```diff
@@ -10,13 +9,6 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
     <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
 
-    <permission
```

Comment:
amazon changes do not relate to this PR, should revert this change in the PR

Reply:
Gotcha, I wanted to change the default from using Amazon to not using Amazon, but I can make a separate PR for that